ArduPlane/quadplane.cpp

```diff
@@ -1116,6 +1116,16 @@ void QuadPlane::control_qacro(void)
     }
 }
 
+void QuadPlane::relax_attitude_control(){
+    // if not a vectored tailsitter completely disable attitude control
+    uint8_t chan;
+    bool vectored_tailsitter = SRV_Channels::find_channel(SRV_Channel::k_tiltMotorLeft, chan) &&
```

Comment:
```suggestion
    bool vectored_tailsitter = SRV_Channels::function_assigned(SRV_Channel::k_tiltMotorLeft)
```

I don't think you have to check for both, no one should have one and not the other

Reply:
I forgot we had a function_assigned method. 
Perhaps we should have a less indirect means of identifying a vectored tailsitter?